doc/mediasdk-man.md

```diff
@@ -1469,6 +1592,97 @@ Global functions initialize and de-initialize the SDK library and perform query
 [MFXGetPriority](#MFXGetPriority_1)    | Obtain session priority
 [MFXDisjoinSession](#MFXDisjoinSession) | Remove the join state of the current session
 [MFXClose](#MFXClose)                  | De-initializes an SDK session
+[MFXQueryAdaptersNumber](#MFXQueryAdaptersNumber) | Queries number of Intel Gen Graphics adapters
+[MFXQueryAdapters](#MFXQueryAdapters)             | Queries information about suitable adapters for given workload
+[MFXQueryAdaptersDecode](#MFXQueryAdaptersDecode) | Queries information about suitable adapters for decode workload
+
+### <a id='MFXQueryAdaptersNumber'>MFXQueryAdaptersNumber</a>
+
+**Syntax**
+
+[mfxStatus](#mfxStatus) `MFXQueryAdaptersNumber(mfxU32* num_adapters);`
+
+**Parameters**
+
+| | |
+--- | ---
+`num_adapters` | Pointer for output number of detected Intel Gen Graphics adapters
+
+**Description**
+
+This function returns number of Intel Gen Graphics adapters. It can be used before [MFXQueryAdapters](#MFXQueryAdapters) call to determine size of input data which user needs to allocate.
+
+**Return Status**
+
+| | |
+--- | ---
+`MFX_ERR_NONE` | The function completed successfully.
+`MFX_ERR_NULL_PTR` | `num_adapters` pointer is NULL.
+
+**Change History**
+
+This function is available since SDK API **TBD**.
+
+### <a id='MFXQueryAdapters'>MFXQueryAdapters</a>
+
+**Syntax**
+
+[mfxStatus](#mfxStatus) `MFXQueryAdapters(mfxComponentInfo* input_info, mfxAdaptersInfo* adapters);`
+
+**Parameters**
+
+| | |
+--- | ---
+`input_info` | Pointer to workload description. See [mfxComponentInfo](#mfxComponentInfo) description for details.
```

Comment:
Can I pass input_info=nullptr? What I will get?

Reply:
list of adapters. If I forgot to mention it in doc, I will add

Reply:
Doc updated